wordpress-mobile/WordPress-iOS
# Time zone settings

Support for changing the time zone in Settings for WordPress.com sites

![ios-timezone-settings](https://user-images.githubusercontent.com/8739/35052007-d72b23d0-fba6-11e7-9a0b-03c08468756b.gif)

To keep things simple, I'm only persisting in memory, and otherwise fetching from the network. If there's a memory warning, the store will reset its state and release the cached time zones.

Fixes #7831 

To test:

- Try to change the time zone in a WordPress.com site and a Jetpack site
- Verify that the change reflects in the web
- Verify that search works as expected
